### PR TITLE
Refactor: Get fragment description through fetching of plain html

### PR DESCRIPTION
**Pull request by jjjaspher**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

- Fetch plain html of first 10 article and get fragment description to be display.
- Removed unused fields on the query-index (lastModified, publisheddate, longdescriptionextracted).

Fix https://github.com/hlxsites/accenture-newsroom/issues/480

Test URLs:
- Before: https://main--accenture-newsroom--hlxsites.hlx.live/
- After: https://fragment-desc--accenture-newsroom--hlxsites.hlx.live/

### CODE REVIEW

**Review by ramboz — APPROVED** [review]
Looks good to me 👍🏼

File: helix-query.yaml [hunk 1]
```diff
@@ -6,18 +6,12 @@ indices:
       - /**
     target: /query-index.json
     properties:
-      lastModified:
```
Comment (shsteimer) [inline helix-query.yaml:9]: I'd recommend not changing the query index file just yet. Instead, I'd probably approach this in phases. basically get all the code changes done, working and deployed to no longer use those fields. Once sure that is good, then change the index definition, the index spreadsheet, and finally run a re-index to bring the index up to date.

**Review by shsteimer — COMMENTED** [review]
in addition to the comment on the query index, a few general thoughts:

* I think long description is used by search. can we change that to just use description instead?
* minor concern on the performance, 94. I'm not sure if that is a degradation from before or not, but I'd start by changing your test urls to use hlx.live not .page (Ideally, we should update the PR template, as that is really what you should always be doing as best practice). If that doesn't help enough, I think we could look at ways to optimize this.

**Comment by shsteimer** [comment]
@jjjaspher I took a closer look at perf and it looks to me to be CLS. I think if we can add a min-height on `.newslist-item` that should remove most if not all of the cls (that height will need to be a bit of a guess, and probably different across breakpoints). 

One option would be to add that min-height only while the description is loading, so something like:
1. add a class to the item, `description-pending` or whatever
2. tie min-height to that class
3. once it's loaded, remove the class, and thus the min-height, allowing the item's height to just be auto

File: helix-query.yaml (hunk 1 above)
Comment (jjjaspher) [reply helix-query.yaml]: helix-query.yaml reverted

**Comment by jjjaspher** [comment]
> * I think long description is used by search. can we change that to just use description instead?

Hi @shsteimer , our description column is not a clean data, it was being cutoff from many articles.
image


For the CLS issue, I implemented the min-height for both newslist card and search result. We're now back to 100 performance score.
Please let me know if there are other concern. Thanks!

**Review by shsteimer — APPROVED** [review]

**Comment by shsteimer** [comment]
@jjjaspher everything looks good and is good to merge IMHO.

I do think we still need to do something about search as that uses the long description from the query index at https://github.com/hlxsites/accenture-newsroom/blob/main/blocks/newslist/newslist.js#L112

but can probably do that in a separate PR

**Pull request closed without merging**